include/pybind11/attr.h

```diff
@@ -414,11 +414,15 @@ inline void check_kw_only_arg(const arg &a, function_record *r) {
         pybind11_fail("arg(): cannot specify an unnamed argument after a kw_only() annotation or args() argument");
 }
 
+inline void check_have_self_arg(function_record *r) {
+    if (r->is_method && r->args.empty())
+        r->args.emplace_back("self", nullptr, handle(), true /*convert*/, false /*none not allowed*/);
```

Comment:
`/*convert=*/ true`
?
Super minor of course, but I don't remember ever seeing `value /*keyword*/`, it flips compared to expectations (Python).

Reply:
Works for me; I didn't actually add this, just moved it as-is.

Reply:
Okay I *did* actually put that comment there, nearly 5 years ago in abc29cad020bb035f698fdc3e5d33c5c551ed772  :flushed:

Reply:
Updated.